examples/air-quality-sensor-app/air-quality-sensor-common/src/air-quality-sensor-manager.cpp

```diff
@@ -136,78 +137,77 @@ void AirQualitySensorManager::Init()
 
 void AirQualitySensorManager::OnAirQualityChangeHandler(AirQualityEnum newValue)
 {
-    mAirQualityInstance.UpdateAirQuality(static_cast<AirQualityEnum>(newValue));
-    ChipLogDetail(NotSpecified, "Updated AirQuality value: %huu", newValue);
+    Status status = mAirQualityInstance.UpdateAirQuality(static_cast<AirQualityEnum>(newValue));
+    VerifyOrReturn(Status::Success == status, ChipLogError(NotSpecified, "Failed to set AirQuality attribute"));
 }
 
 void AirQualitySensorManager::OnCarbonDioxideMeasurementChangeHandler(float newValue)
 {
-    mCarbonDioxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
-    ChipLogDetail(NotSpecified, "Updated Carbon Dioxide: %f", newValue);
+    ChipError chipError = mCarbonDioxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
+    VerifyOrReturn(CHIP_NO_ERROR == chipError, ChipLogError(NotSpecified, "Failed to set CarbonDioxide MeasuredValue attribute"));
 }
 
 void AirQualitySensorManager::OnCarbonMonoxideMeasurementChangeHandler(float newValue)
 {
-    mCarbonMonoxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
-    ChipLogDetail(NotSpecified, "Updated Carbon Monoxide value: %f", newValue);
+    ChipError chipError = mCarbonMonoxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
+    VerifyOrReturn(CHIP_NO_ERROR == chipError, ChipLogError(NotSpecified, "Failed to set CarbonMonoxide MeasuredValue attribute"));
 }
 
 void AirQualitySensorManager::OnNitrogenDioxideMeasurementChangeHandler(float newValue)
 {
-    mNitrogenDioxideConcentrationMeasurementInstance.SetMeasuredValue(MakeNullable(newValue));
-    ChipLogDetail(NotSpecified, "Updated Nitrogen Dioxide value: %f", newValue);
```

Comment:
Why are all these detail logs logging the new value being removed?

Reply:
See answer above https://github.com/project-chip/connectedhomeip/pull/29329/files#r1335751303